opengrok-indexer/src/test/java/org/opengrok/indexer/authorization/AuthorizationFrameworkReloadTest.java

```diff
@@ -153,7 +146,7 @@ public void run() {
         }
 
         // Double check that at least one reload() was done.
-        reloads = stats.getRequest("authorization_stack_reload") - reloads;
+        long reloads = Metrics.getInstance().counter("authorization_stack_reload").getCount();
```

Comment:
doesn't it interfere with other tests? the previous version expected a change in reloads